README.md

```diff
@@ -52,7 +57,8 @@ sudo dnf install ghdl gtkwave
     ```
 
     **Note:** The AppImage does not provide a GUI, so double-clicking it does nothing.
-
+    
+    **Note:** For the Arch make sure to install [AppImageLauncher<sup>AUR</sup>](https://aur.archlinux.org/packages/appimagelauncher) first.
```

Comment:
Parvaj is terminal-based, so no need for AppImageLauncher.

Reply:
For the AppImage way to install and run, it is needed.
I tested without that package and I got this:
```bash
exec: Failed to execute process '/home/koushan/.local/bin/parvaj': The file exists and is executable. Check the interpreter or linker?
```

Reply:
Ah. That's unfortunate. Had you installed required libraries for running AppImages (like fuse)?

BTW, I'll test it in an Arch Docker container.

Reply:
I've checked it. At least for Arch Linux's Docker container, there is no need for having AppImageLauncher.

Generally, AppImages are self-contained, so they doesn't rely on things like AppImageLauncher.